### ISSUE TITLE
Close unexpectedly on local playback

### ISSUE THREAD

**Issue by armageddon421** [badaix/snapdroid#9]
When I press play at the top the App closes unexpectedly. The music is then playing perfectly fine while that error dialog ist still open but obviously stops as soon as I choose to close or restart the app.
This is on a oneplus 6 with Android 8.1.0/Oxygen OS 5.1.5

**Comment by armageddon421** [comment]
This happened with the version 0.14.0 from the GitHub releases. I just tried the play store version (it reports 0.14.0.14) and there everything seems fine.

**Comment by maweki** [comment]
It happened for me with both 0.13 and 0.14 with Oreo.

**Comment by badaix** [comment]
I think it's broken since I've updated the SDK version to 26: f422fb7.
I guess that the crash is related to the updated notification API, which unfortunately seems not to be solved with 872350b (as far as I remember there was a bug report with a stack trace and the Android Linter was also pointing me to the notification API).
Can you please provide a stack trace of the crash? Unfortunately, I cannot reproduce the bug with the emulator, and my latest phone is running Marshmallow.
Also I wasn't aware of a Playstore version. I found [this one](https://play.google.com/store/apps/details?id=uk.org.jaggard.snapcast), which seems to be from [this fork](https://github.com/mjaggard/snapcast), and this fork is still at SDK version 25, so it shouldn't be affected from notification crash.

**Comment by pizzaboy75** [comment]
This is what i get on 8.1 with a logcat running during the crash:

```
05-29 09:33:32.816  3124  6050 D audio_route: Apply path: speaker-protected
05-29 09:33:32.823  3124  6050 D audio_hw_primary: enable_snd_device: snd_device(95: vi-feedback)
05-29 09:33:32.823  3124  6050 D audio_route: Apply path: vi-feedback
05-29 09:33:32.823  3124  6050 D audio_hw_primary: enable_audio_route: usecase(24) apply and update mixer path: spkr-vi-record
05-29 09:33:32.823  3124  6050 D audio_route: Apply path: spkr-vi-record
05-29 09:33:32.844  3537  3570 W ActivityManager: Error showing notification for service
05-29 09:33:32.844  3537  3570 W ActivityManager: java.lang.RuntimeException: invalid channel for service notification: Notification(channel=default_notification_channel_id pri=0 contentView=null vibrate=null sound=null tick defaults=0x0 flags=0x40 color=0x00000000 actions=1 vis=PRIVATE)
05-29 09:33:32.844  3537  3570 W ActivityManager: 	at com.android.server.am.ServiceRecord$1.run(ServiceRecord.java:532)
05-29 09:33:32.844  3537  3570 W ActivityManager: 	at android.os.Handler.handleCallback(Handler.java:790)
05-29 09:33:32.844  3537  3570 W ActivityManager: 	at android.os.Handler.dispatchMessage(Handler.java:99)
05-29 09:33:32.844  3537  3570 W ActivityManager: 	at android.os.Looper.loop(Looper.java:164)
05-29 09:33:32.844  3537  3570 W ActivityManager: 	at android.os.HandlerThread.run(HandlerThread.java:65)
05-29 09:33:32.844  3537  3570 W ActivityManager: 	at com.android.server.ServiceThread.run(ServiceThread.java:46)
05-29 09:33:32.847  3537  5921 I WifiService: acquireWifiLock uid=10080 lockMode=3
05-29 09:33:32.851  3124  6050 D audio_hw_primary: enable_audio_route: usecase(1) apply and update mixer path: low-latency-playback
05-29 09:33:32.851  3124  6050 D audio_route: Apply path: low-latency-playback
05-29 09:33:32.852  1525  1525 D AndroidRuntime: Shutting down VM
05-29 09:33:32.852  1525  1525 E AndroidRuntime: FATAL EXCEPTION: main
05-29 09:33:32.852  1525  1525 E AndroidRuntime: Process: de.badaix.snapcast, PID: 1525
05-29 09:33:32.852  1525  1525 E AndroidRuntime: android.app.RemoteServiceException: Bad notification for startForeground: java.lang.RuntimeException: invalid channel for service notification: Notification(channel=default_notification_channel_id pri=0 contentView=null vibrate=null sound=null tick defaults=0x0 flags=0x40 color=0x00000000 actions=1 vis=PRIVATE)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1768)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6494)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
05-29 09:33:32.852  1525  1525 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
05-29 09:33:32.856  3537  6051 W ActivityManager:   Force finishing activity de.badaix.snapcast/.MainActivity
05-29 09:33:32.862  3537  3571 I ActivityManager: Showing crash dialog for package de.badaix.snapcast u0
05-29 09:33:32.889  3124  6050 E msm8974_platform: ramp_speaker_gain: Could not get ctl for mixer cmd - Left Speaker Gain or Right Speaker Gain, not applying speaker gain ramp
05-29 09:33:32.889  3124  6050 D audio_hw_primary: out_write: retry previous failed cal level set
05-29 09:33:32.889  3124  6050 W msm8974_platform: platform_send_gain_dep_cal: Usecase list is empty
05-29 09:33:32.896  3537  8030 I OpenGLRenderer: Initialized EGL, version 1.4
05-29 09:33:32.896  3537  8030 D OpenGLRenderer: Swap behavior 2
05-29 09:33:33.358  3537  3570 W ActivityManager: Activity pause timeout for ActivityRecord{416b287 u0 de.badaix.snapcast/.MainActivity t37454 f}
05-29 09:33:33.366  3124  3527 D audio_hw_primary: disable_audio_route: usecase(1) reset and update mixer path: low-latency-playback
05-29 09:33:33.380  3124  3527 D audio_hw_primary: disable_snd_device: snd_device(95: vi-feedback)
05-29 09:33:33.380  3124  3527 D audio_hw_primary: disable_audio_route: usecase(24) reset and update mixer path: spkr-vi-record

**Comment by pizzaboy75** [comment]
Yes, new version is silent. Working as expected on Android 8.1. Thanks!

**Comment by badaix** [comment]
fixed in release v0.15.0
